game/net/server/game-data-handler.js

```diff
@@ -75,6 +75,118 @@ function handleDisconnectVoice(data, conn) {
   conn.send(packet);
 }
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
Need to change how packet handling is done here - Too many subcommands under a single opcode